lib/dry/web/roda/templates/subapp/view__context.rb.tt

```diff
@@ -2,7 +2,7 @@ require "<%= config[:underscored_umbrella_name] %>/view/context"
 
 module <%= config[:camel_cased_umbrella_name] %>
   module <%= config[:camel_cased_app_name] %>
-    module View
+    class View
```

Comment:
actually this doesn't work because we get a superclass mismatch. :frowning_face:

Reply:
Yep, this is very annoying. You have to `class View < Dry::View` again.  IMHO, this is because https://github.com/dry-rb/dry-view/issues/65 was a bad decision, but with `1.0` out there surely it is too late to regret. In my project, as a workaround, I have named my view `HtmlView`, so I can use `module View` for context and others. But I don't like it, neither.

Reply:
Thanks for that link - yeah, I've had to call it `View::BaseView` here because to me redefining the superclass (and in fact using a class for the namespace) is not okay.